code/modules/antagonists/changeling/changeling.dm

```diff
@@ -6,10 +6,10 @@
 	var/datum/abilityHolder/changeling/ability_holder
 
 	is_compatible_with(datum/mind/mind)
-		return ishuman(mind.current)
+		return isliving(mind.current)
```

Comment:
I have-ish, the HUD doesn't work, so you can't use them, so they can't break.

Even so, it would require admin intervention to put a changeling datum onto a critter.